src/components/OurTeam/OurTeam.jsx

```diff
@@ -0,0 +1,5 @@
+const OurTeam = () => {
+  return <div>Our Team</div>;
```

Comment:
place this file in the board folder instead, make sure to git pull

Reply:
Have these folders been created? I pulled but cannot seem to find them.